flagmap.go

```diff
@@ -64,6 +64,23 @@ func (f FlagMap) String(long string) string {
 	return s
 }
 
+// StringSlice returns the given flag value as string slice.
+// Panics if not present. Flags must be registered.
+func (f FlagMap) StringSlice(long string) []string {
+	i := f[long]
+	if i == nil {
+		panic(fmt.Errorf("missing flag value: flag '%s' not registered", long))
+	}
+	s := make([]string, 0)
```

Comment:
cache the length of `i.Value.([]interface{})` and initialize the slice with the length.